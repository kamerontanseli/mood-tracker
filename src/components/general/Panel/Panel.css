.Panel {
  padding: 20px;
  background-color: #fff;
  border: 1px solid #eee;
  border-radius: 4px;
  box-shadow: 0px 2px 5px -1px #eee;
}