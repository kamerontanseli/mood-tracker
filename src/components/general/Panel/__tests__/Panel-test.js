import React from 'react'
import renderer from 'react-test-renderer'
import Panel from '..'

describe('render', () => {
  it('should render', () => {
    const snapshot = renderer.create(
      <Panel>Hello world</Panel>
    );
    expect(snapshot).toMatchSnapshot()
  })
})