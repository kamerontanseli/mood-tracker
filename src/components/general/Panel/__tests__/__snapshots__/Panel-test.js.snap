// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render should render 1`] = `
<div
  className="Panel"
>
  Hello world
</div>
`;
